courseraprogramming/utils.py

```diff
@@ -129,13 +129,9 @@ def docker_client(args):
                 timeout=args.timeout,
                 **kwargs_from_env())
     elif _platform == 'darwin':
-        # OS X - Assume boot2docker, and pull from that environment.
+        # OS X.
         kwargs = kwargs_from_env()
-        if len(kwargs) == 0:
-            logging.error('Could not correctly pull in docker environment. '
```

Comment:
what happens if we're running the older docker-machine and the environment is not set correctly?